app/src/main/java/com/example/ithaca_transit_android_v2/models/DirectionType.kt

```diff
@@ -1,9 +1,13 @@
 package com.example.ithaca_transit_android_v2.models
 
+import com.squareup.moshi.Json
+
 /*
 Represents the type of route that is displayed on the map
  */
-enum class RouteType {
+enum class DirectionType {
```

Comment:
is this tested? i'm not tried to use `@Json` this way before...

Reply:
When the body was printed in logcat, the correct enums were being identified